### PR TITLE
Edit the style of README.md and example

### PR DESCRIPTION
**Pull request by poozhu**
Add Travis CI (Maybe useless ),  
Open in browser.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,19 @@
-# 𝑾𝒆𝒊𝒓𝒅 𝑭𝒐𝒏𝒕𝒔
-𝑨 𝑱𝒂𝒗𝒂𝑺𝒄𝒓𝒊𝒑𝒕 𝒑𝒂𝒄𝒌𝒂𝒈𝒆 𝒕𝒉𝒂𝒕 𝒕𝒖𝒓𝒏 𝒂𝒍𝒑𝒉𝒂𝒏𝒖𝒎𝒆𝒓𝒊𝒄 𝒄𝒉𝒂𝒓𝒂𝒄𝒕𝒆𝒓𝒔 𝒊𝒏𝒕𝒐 𝒘𝒆𝒊𝒓𝒅 𝒇𝒐𝒏𝒕 𝒔𝒕𝒚𝒍𝒆.
+# 𝖂𝖊𝖎𝖗𝖉 𝕱𝖔𝖓𝖙𝖘
```
Comment (beizhedenglong) [inline README.md]: Hi, why change the font style?  I think this font style is hard to read.

Comment (poozhu) [reply README.md]: More classical or weirder ? but that's just my opinions 🎲

Comment (beizhedenglong) [reply README.md]: Can you change it to original style?

Comment (poozhu) [reply README.md]: about Readme.md ?

Comment (beizhedenglong) [reply README.md]: yes

Comment (poozhu) [reply README.md]: I also use this font on the example page and the img of demo have changed too.  Is that OK ?

Comment (beizhedenglong) [reply README.md]: That's fine. Thanks.

**Pull request merged**